### PR TITLE
Typo "patch**\&#42;**"→"patch*"

### PR DESCRIPTION
**Pull request by hyoshioka0128**
https://docs.microsoft.com/en-us/sql/sql-server/sql-server-2012-release-notes?view=sql-server-2016
https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/sql-server/sql-server-2012-release-notes.md
#PingMSFTDocs

### CODE REVIEW

**Comment by PRMerger15** [comment]
@hyoshioka0128 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Jak-MS** [comment]
@MikeRayMSFT 

Can you review this PR?

IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by PRMerger8** [comment]
Invalid command: '#label'. Only Microsoft employees can use this command.

File: docs/sql-server/sql-server-2012-release-notes.md [hunk 1]
```diff
@@ -506,13 +506,13 @@ The following table summarizes driver support for Always On Availability Groups:
 |----------|--------------------------|----------------------|----------------------|------------------------------------------------------------------|---------------------------------------------------------------------------------|  
 |SQL Native Client 11.0 ODBC|Yes|Yes|Yes|Yes|Yes|  
 |SQL Native Client 11.0 OLEDB|No|Yes|Yes|No|No|  
-|ADO.NET with .NET Framework 4.0 with connectivity patch**\&#42;**|Yes|Yes|Yes|Yes|Yes|  
-|ADO.NET with .NET Framework 3.5 SP1 with connectivity patch **\&#42;\&#42;**|Yes|Yes|Yes|Yes|Yes|  
+|ADO.NET with .NET Framework 4.0 with connectivity patch*|Yes|Yes|Yes|Yes|Yes|
```
Comment (rwestMSFT) [inline docs/sql-server/sql-server-2012-release-notes.md:509]:
```suggestion
|ADO.NET with .NET Framework 4.0 with connectivity patch *|Yes|Yes|Yes|Yes|Yes|  
```

File: docs/sql-server/sql-server-2012-release-notes.md [hunk 2]
```diff
@@ -506,13 +506,13 @@ The following table summarizes driver support for Always On Availability Groups:
 |----------|--------------------------|----------------------|----------------------|------------------------------------------------------------------|---------------------------------------------------------------------------------|  
 |SQL Native Client 11.0 ODBC|Yes|Yes|Yes|Yes|Yes|  
 |SQL Native Client 11.0 OLEDB|No|Yes|Yes|No|No|  
-|ADO.NET with .NET Framework 4.0 with connectivity patch**\&#42;**|Yes|Yes|Yes|Yes|Yes|  
-|ADO.NET with .NET Framework 3.5 SP1 with connectivity patch **\&#42;\&#42;**|Yes|Yes|Yes|Yes|Yes|  
+|ADO.NET with .NET Framework 4.0 with connectivity patch*|Yes|Yes|Yes|Yes|Yes|  
+|ADO.NET with .NET Framework 3.5 SP1 with connectivity patch**|Yes|Yes|Yes|Yes|Yes|
```
Comment (rwestMSFT) [inline docs/sql-server/sql-server-2012-release-notes.md:510]:
```suggestion
|ADO.NET with .NET Framework 3.5 SP1 with connectivity patch **|Yes|Yes|Yes|Yes|Yes|  
```

File: docs/sql-server/sql-server-2012-release-notes.md [hunk 3]
```diff
@@ -506,13 +506,13 @@ The following table summarizes driver support for Always On Availability Groups:
...
 |SQL Native Client 11.0 ODBC|Yes|Yes|Yes|Yes|Yes|  
 |SQL Native Client 11.0 OLEDB|No|Yes|Yes|No|No|  
-|ADO.NET with .NET Framework 4.0 with connectivity patch**\&#42;**|Yes|Yes|Yes|Yes|Yes|  
-|ADO.NET with .NET Framework 3.5 SP1 with connectivity patch **\&#42;\&#42;**|Yes|Yes|Yes|Yes|Yes|  
+|ADO.NET with .NET Framework 4.0 with connectivity patch*|Yes|Yes|Yes|Yes|Yes|  
+|ADO.NET with .NET Framework 3.5 SP1 with connectivity patch**|Yes|Yes|Yes|Yes|Yes|  
 |Microsoft JDBC driver 4.0 for SQL Server|Yes|Yes|Yes|Yes|Yes|  
   
-**\&#42;** Download the connectivity patch for ADO .NET with .NET Framework 4.0: [https://support.microsoft.com/kb/2600211](https://support.microsoft.com/kb/2600211).  
+*Download the connectivity patch for ADO .NET with .NET Framework 4.0: [https://support.microsoft.com/kb/2600211](https://support.microsoft.com/kb/2600211).
```
Comment (rwestMSFT) [inline docs/sql-server/sql-server-2012-release-notes.md:513]:
```suggestion
* Download the connectivity patch for ADO .NET with .NET Framework 4.0: [https://support.microsoft.com/kb/2600211](https://support.microsoft.com/kb/2600211).  
```

File: docs/sql-server/sql-server-2012-release-notes.md [hunk 4]
```diff
@@ -506,13 +506,13 @@ The following table summarizes driver support for Always On Availability Groups:
...
-|ADO.NET with .NET Framework 3.5 SP1 with connectivity patch **\&#42;\&#42;**|Yes|Yes|Yes|Yes|Yes|  
+|ADO.NET with .NET Framework 4.0 with connectivity patch*|Yes|Yes|Yes|Yes|Yes|  
+|ADO.NET with .NET Framework 3.5 SP1 with connectivity patch**|Yes|Yes|Yes|Yes|Yes|  
 |Microsoft JDBC driver 4.0 for SQL Server|Yes|Yes|Yes|Yes|Yes|  
   
-**\&#42;** Download the connectivity patch for ADO .NET with .NET Framework 4.0: [https://support.microsoft.com/kb/2600211](https://support.microsoft.com/kb/2600211).  
+*Download the connectivity patch for ADO .NET with .NET Framework 4.0: [https://support.microsoft.com/kb/2600211](https://support.microsoft.com/kb/2600211).  
   
-**\&#42;\&#42;** Download the connectivity patch for ADO.NET with .NET Framework 3.5 SP1: [https://support.microsoft.com/kb/2654347](https://support.microsoft.com/kb/2654347).  
+**Download the connectivity patch for ADO.NET with .NET Framework 3.5 SP1: [https://support.microsoft.com/kb/2654347](https://support.microsoft.com/kb/2654347).
```
Comment (rwestMSFT) [inline docs/sql-server/sql-server-2012-release-notes.md:515]:
```suggestion
** Download the connectivity patch for ADO.NET with .NET Framework 3.5 SP1: [https://support.microsoft.com/kb/2654347](https://support.microsoft.com/kb/2654347).  
```

**Comment by rwestMSFT** [comment]
@hyoshioka0128 Thank you for your contribution. If you accept the suggested edits, we'll get this merged in the next few days.

**Comment by PRMerger4** [comment]
@hyoshioka0128 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by hyoshioka0128** [comment]
@rwestMSFT - Thanks!

**Comment by rwestMSFT** [comment]
We found inconsistencies elsewhere in the article, so I created an internal PR that corrected those. Thank you for your feedback and for helping improve content, even if it is ten years old.

**Pull request closed without merging**